### PR TITLE
Round trip XPC Codable tests

### PR DESCRIPTION
**Pull request by amomchilov**
Builds on top of https://github.com/trilemma-dev/SecureXPC/pull/8 , most commits are in common. Merge that one first.

### CODE REVIEW

File: Tests/SecureXPCTests/XPC Codable round trip tests.swift [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-07.
+//
+
+import XCTest
+@testable import SecureXPC
+
+class SampleClass: Codable {
+	// Properies need to be mutable, otherwise you get this warning:
```
Comment (jakaplan) [inline Tests/SecureXPCTests/XPC Codable round trip tests.swift:12]: Alternatively make these `let` and don't assign them initial values? Having them be `var` values is fine though, just not sure if your comment is a way of indicating you find this to be a workaround of some sort.

Comment (jakaplan) [reply Tests/SecureXPCTests/XPC Codable round trip tests.swift:12]: Yup, I think this is fine as-is.

**Comment by jakaplan** [comment]
This looks good to go; will merge this after #8 is ready to be merged.

**Comment by amomchilov** [comment]
By the way, you can re-review and just click "approve" and it'll show that :)

I'll try to iron out #8 today, but I've been busy

**Pull request merged**